core/domain/html_validation_service.py

```diff
@@ -852,6 +855,118 @@ def add_dimensions_to_image_tags(exp_id, html_string):
     return python_utils.UNICODE(soup).replace('<br/>', '<br>')
 
 
+def generate_math_svgs_filename(height, width, vertical_padding):
+    """Generates a random filename for a math SVG based on the given
+    dimensions.
+
+    Args:
+        height: str. Height of the image.
+        width: str. Width of the image.
+        vertical_padding: str. The vertical padding required for the SVG when
+            displayed inline.
+
+    Returns:
+        str. The name of the SVG file with its dimensions in it.
+    """
+
+    random_string = (
+        ''.join(
+            random.choice(
+                string.ascii_lowercase + string.digits) for _ in (
+                    python_utils.RANGE(10))))
+    date_object = datetime.datetime.utcnow()
+    year = python_utils.UNICODE(date_object.year)
+    month = python_utils.UNICODE(date_object.month)[-2:]
+    day = python_utils.UNICODE(date_object.day)[-2:]
+    hour = python_utils.UNICODE(date_object.hour)[-2:]
+    minute = python_utils.UNICODE(date_object.minute)[-2:]
+    second = python_utils.UNICODE(date_object.second)[-2:]
+    date_time_string = (
+        year + month + day + hour + minute + second + random_string)
+
+    filename = (
```

Comment:
Use string interpolation please -- this is a convention in this codebase.

Reply:
done